examples/fit_with_bounds.py

```diff
@@ -36,6 +36,7 @@ def gen_from_expr(expr, guesses={}):
     """
     # {{{ decide which symbols are parameters vs. variables
     all_symbols = expr.atoms(sp.Symbol)
+    # you need to pass "empty_data" to the function to enable the following line to work
```

Comment:
this needs to be fixed

Reply:
added definition of empty data inside function